### DISCUSSION TITLE
지금 제페토 접속 저만 안되나요?

### DISCUSSION THREAD

**Discussion by HJ3150** [naverz/zepeto-studio-kor#320] [4. General]
멀티플레이 월드맵 개발 공부 하는 도중에 유니티 에디터에서 플레이도 정상적으로 안되고
스마트폰 앱으로도 접속이 안되네요

**Comment by minjujuu** [comment]
지금은 접속 되는 것 같아요. 다시 한 번 접속 해보실래요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

일시적인 서버 장애로 예상됩니다.

불편을 드려서 죄송합니다.
